wpimath/src/main/java/edu/wpi/first/math/controller/PIDController.java

```diff
@@ -303,8 +302,7 @@ public double getVelocityError() {
    * @return The next controller output.
    */
   public double calculate(double measurement, double setpoint) {
-    // Set setpoint to provided value
-    setSetpoint(setpoint);
+    m_setpoint = setpoint;
```

Comment:
Why does this no longer call setSetpoint(), especially given that the functionality of atSetpoint moved to setSetpoint()?  It seems like this would be a breaking change to code which calls calculate() and atSetpoint()?

Reply:
Because this calculate() delegates to one that updates the error variables. Calling setSetpoint() would make this function compute the error twice.